src/lib/parser/cli.ts

```diff
@@ -197,7 +197,20 @@ function clearScreen () {
  * Watch mode - continuously monitor and update display
  */
 async function watchMode (input: string | MgbaWebSocketClient, options: { debug: boolean, graph: boolean, interval: number }) {
```

Comment:
lets refactor this file to have its watch code as simple and concise as possible, since PokemonSaveParser should do all the heavy lifting